tests/unit/test_load.py

```diff
@@ -0,0 +1,19 @@
+import os
+from datetime import datetime
+
+from google.cloud.storage.bucket import Bucket
+
+
+def test_load_get_bucket(load):
+    assert type(load._get_bucket()) == Bucket
+    assert load._get_bucket().name == os.getenv("BUCKET")
```

Comment:
should we mock os? I mean is it reliable?

Reply:
I made some updates to the use of `os` that will come through in next couple PRs. Also, I want to say it should be reliable cause will be inside of container and env variables passed in through `docker-compose`. I haven't implemented it yet but I think it'll make more sense once everything is in place.

Reply:
thanks, good point